web-components/src/components/modal/FormModalTemplate.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import { makeStyles } from '@mui/styles';
+import Modal, { RegenModalProps } from '../modal';
+import Title from '../title';
+
+const useStyles = makeStyles(theme => ({
+  modal: {
+    height: 'inherit',
```

Comment:
Is this necessary? It makes the modal stretch higher than the screen if the content is too tall.
```suggestion
```

Reply:
this is how the put modal looks like without it:
<img width="1498" alt="image" src="https://user-images.githubusercontent.com/3149358/161214989-062389df-48f2-4370-9b53-cabd2e5c6f2b.png"> 
which isn't what we want

Reply:
Ok, we can leave this. I have some other modal tweaks that will satisfy all cases.